code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1860,6 +1860,17 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 		return FALSE
 	if(attacker_style && attacker_style.disarm_act(user,target))
 		return TRUE
+
+	if(!(target.mobility_flags & MOBILITY_STAND) && (user.zone_selected == BODY_ZONE_L_LEG || user.zone_selected == BODY_ZONE_R_LEG) && user.a_intent == INTENT_DISARM && !target.shoes)
```

Comment:
if you're actually serious about this, then this fails to compile because we don't have mobility flags.
```suggestion
	if(target.lying && (user.zone_selected == BODY_ZONE_L_LEG || user.zone_selected == BODY_ZONE_R_LEG) && user.a_intent == INTENT_DISARM && !target.shoes)
```